backend/server.ts

```diff
@@ -32,8 +32,8 @@ app.get('/test', async (req, res: Response<any, number>) => {
   await Sender.sendMessages(res);
   await Sender.sendLineCounts(res);
   await Sender.sendTopWords(res);
+  await Sender.sendDuccScores(res); // Daily
 
-  // Daily
   setInterval(async (_) => {
```

Comment:
I assume these are auto formatter changes. In this case it doesn't really make sense. Maybe move the comment next to line 37 so it's satisfied.